command/format/diff.go

```diff
@@ -518,6 +526,12 @@ func (p *blockBodyDiffPrinter) writeNestedBlockDiff(name string, label *string,
 }
 
 func (p *blockBodyDiffPrinter) writeValue(val cty.Value, action plans.Action, indent int) {
+	// Could check specifically for the sensitivity marker
+	if val.IsMarked() {
+		p.buf.WriteString("(sensitive)")
```

Comment:
I don't know anything about marks, so please excuse the question if it's nonsense: 

Is there any risk that not checking for the specific sensitive marker here could cause problems in the future? Is there any chance that a provider could start using marks, and then someone using terraform 0.14 and that provider would run into a problem?

Reply:
There's not a risk of that. `Value`s can't be encoded with marks, so a provider cannot send marked values, and that's also why we have to unmark/remark values sending them through providers.

On the methods themselves, `IsMarked` is a very simple check, with `HasMark` being the more specific one. https://github.com/zclconf/go-cty/blob/master/cty/marks.go#L90-L102 Using `HasMark` would require creating a type specific to the mark. Since there's only one (naive) mark presently ("sensitive"), this is avoiding that at the moment.